### PR TITLE
doc: add Turing RK1 docs to Single Board Computer section

### PR DESCRIPTION
**Pull request by nberlee**
This add documentation on how to setup the Turing RK1 with Talos the first time.

Booting can be done with eMMC or NVMe (using a U-Boot SPI image on eMMC)

This commit also add Turing RK1 to the SBC support matrix.

### CODE REVIEW

File: website/content/v1.9/talos-guides/install/single-board-computers/turing_rk1.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+      ```bash
+      sudo dd if=metal-arm64.raw of=/dev/nvme0n1
+      ```
+3. Find the latest `sbc-rockchip` overlay, download and extract the SBC overlay image:
+    
+     - Find the latest release tag of the [sbc-rockchip repo](https://github.com/siderolabs/sbc-rockchip/releases).
+      - Download the sbc overlay image and extract the SPI image:
+    
+      ```bash
+      mkdir _out
```
Comment (frezbo) [inline website/content/v1.9/talos-guides/install/single-board-computers/turing_rk1.md:80]: probably change this to use `crane` to align with existing docs, check the jetson docs for an example

Comment (nberlee) [reply website/content/v1.9/talos-guides/install/single-board-computers/turing_rk1.md]: done

**Review by frezbo — APPROVED** [review]

**Comment by nberlee** [comment]
Sorry @frezbo 

I did not know there was such a thing as a markdown-linter, otherwise I would have `make lint` before.

**Pull request merged**